### PR TITLE
Fixes inconsistencies between List implementations.

### PR DESCRIPTION
**Pull request by rcorral**

### CODE REVIEW

File: src/js/structs/ServicePlanPhases.js [hunk 1]
```diff
@@ -35,4 +32,6 @@ class ServicePlanPhases extends List {
 
 }
 
+ServicePlanPhases.type = ServicePlanPhase;
```
Comment (rcorral) [inline src/js/structs/ServicePlanPhases.js]: @MatApple did I do this correctly for this file?

**Comment by mesosphere-web** [comment]
Reviewer: @kennyt

File: src/js/structs/ServicePlanPhases.js (hunk 1 above)
Comment (MatApple) [inline src/js/structs/ServicePlanPhases.js]: Yes 👍

**Comment by kennyt** [comment]
good stuff

**Pull request merged**